buildenv/jenkins/jobs/infrastructure/Promote-OpenJ9-OMR-master-to-openj9

```diff
@@ -0,0 +1,64 @@
+/*******************************************************************************
+ * Copyright (c) 2018, 2018 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+
+HTTP = "https://"
+REPO = 'github.com/eclipse/openj9-omr.git'
+TARGET_BRANCH = "openj9"
+
+node('worker') {
+    stage('Promote') {
+        timestamps {
+
+            git branch: "${TARGET_BRANCH}", url: "${HTTP}${REPO}"
+
+            OLD_SHA = sh (
+                script: 'git rev-parse --short HEAD',
+                returnStdout: true
+            ).trim()
+
+            sh "git merge ${OMR_COMMIT} -m 'Merge ${OMR_COMMIT} into ${OLD_SHA}'"
+            
+            MERGE_DATE = sh (
+                script: 'echo `date +%Y%m%d`_`date +%H%M%S`',
+                returnStdout: true
+            ).trim()
+
+            sh """git tag -a 'omr_merge_${MERGE_DATE}'
+                -m 'OMR: ${OMR_COMMIT}'
+                -m 'OpenJ9: ${OPENJ9_COMMIT}'
+                -m 'OpenJDK8: ${OPENJDK8_COMMIT}'
+                -m 'OpenJDK9: ${OPENJDK9_COMMIT}'"""
+
```

Comment:
@DanHeidinga Do we want to continue laying these tags every time we merge?
https://github.com/eclipse/openj9-omr/tags